config/redirects.yml

```
@@ -382,6 +383,9 @@ redirects:
   "/blog/tag/school-experience":  "/train-to-be-a-teacher/get-school-experience"
   "/blog/tag/teaching-internships": "/train-to-be-a-teacher/teaching-internships"
   "/blog/tag/advisers": "/teacher-training-advisers"
+  "/blog/tag/career-change-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
+  "/blog/tag/changing-careers-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
```

```suggestion
  "/blog/tag/changing-career-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
```
current tag is singular re careers!